platform/linux-generic/include/odp/api/plat/pool_types.h

```diff
@@ -48,6 +48,19 @@ typedef enum odp_pool_type_t {
 
 #endif
 
+/** @internal Pool field accessor */
+#define _odp_pool_get(pool, cast, field) \
+	(*(cast *)(uintptr_t)((uint8_t *)pool + _odp_pool_inline.field))
+
+/** @internal Pool header field offsets for inline functions */
+typedef struct _odp_pool_inline_offset_t {
+	/** @internal field offset */
+	uint16_t pool_hdl;
+	/** @internal field offset */
+	uint16_t uarea_size;
+
+} _odp_pool_inline_offset_t;
+
```

Comment:
And this to pool_inline_types.h.

Reply:
I followed the same pattern as the packet inlines. _odp_packet_inline_offset_t is defined in packet_types.h.

Reply:
@matiaselo I agree this is correctly placed here.